portfolio.js

```diff
@@ -1,5 +1,7 @@
 var projects = [];
 
+var portfolio = {};
+
 function Project (title, dateCreated, image, fileSource) {
   this.title = title;
   this.dateCreated = dateCreated;
```

Comment:
Generally, when we make a constructor, we only want it to do one thing - make object instances for us. We don't need to push our project into our projects array because we are actually doing that further down in this file (in a forEach loop).